internal/exec/exec_test.go

```diff
@@ -207,14 +207,14 @@ func TestPseudoConsolePowershell(t *testing.T) {
 	}()
 
 	cmd := "echo \"howdy from conpty\"\r\n"
```

Comment:
Small nit, can we make `"howdy from conpty"` a const and use that on line 201 and format it into this string?

Reply:
Sure!